osquery/logger/plugins/tls.h

```diff
@@ -82,19 +67,10 @@ class TLSLoggerPlugin : public LoggerPlugin {
   Status logStatus(const std::vector<StatusLogLine>& log) override;
 
  private:
-  /**
-   * @brief Hold an auto-incrementing offset for buffered logs.
-   *
-   * Logs are buffered to a backing store until they can be flushed to a TLS
-   * endpoint (based on latency/retry/etc options). Buffering uses a UNIX time
-   * second precision for indexing and ordering. log_index_ helps prevent
-   * collisions by appending an auto-increment counter.
-   */
-  size_t log_index_{0};
+  // Forwarder that buffers/sends logs. Runs in a Dispatcher thread.
+  std::shared_ptr<TLSLogForwarder> forwarder_ = nullptr;
```

Comment:
Not sure what the pros/cons are, but we've typically stuck to initializer list syntax for member variable initialization:
```cpp
std::shared_ptr<TLSLogForwarder> forwarder_{nullptr};
```

Also, if you use `///` the comment is assumed to be the `@breif` string for doxygen.